company-profiles/appquality.md

```diff
@@ -0,0 +1,29 @@
+# AppQuality
+
+## Company blurb
+
+Enable human-centered business solutions unleashing the power of the crowd
+
+## Company size
+
+50+ employees
+
+## Remote status
+
+N/A
+
+## Region
+
+Worldwide
+
+## Company technologies
+
+N/A
```

Comment:
Does the company not use technologies? If, for example, a developer was interested in this company, what programming languages would they need to know?